pyproject.toml

```diff
@@ -95,7 +95,7 @@ replacement = 'src="https://raw.githubusercontent.com/encode/httpx/master/\1"'
 
 # https://beta.ruff.rs/docs/configuration/#using-rufftoml
 [tool.ruff]
-select = ["E", "F", "I", "B", "PIE"]
+select = ["E", "F", "I", "B", "PIE", "C416"]
```

Comment:
I'm neither for or against these code changes, it's probably a neater style yeah.

Tho I'd prefer we stick with minimal confic where possible, so perhaps we can aim for a minimal `select` hete.